statistics_msgs/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>statistics_msgs</name>
+  <version>0.0.0</version>
+  <description>Message definition to publish system metrics topic statistics.</description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
```

Comment:
This seems like a copy-n-paste mistake.

Reply:
I wasn't sure what to put here. Can this package be maintained by ROS Tooling WG while other packages in this repo are maintained by OSRF?

Reply:
Sure.